configs/common/CacheConfig.py

```diff
@@ -192,7 +191,7 @@ def config_cache(options, system):
             # on these names.  For simplicity, we would advise configuring
             # it to use this naming scheme; if this isn't possible, change
             # the names below.
-            if get_runtime_isa() in [ISA.X86, ISA.ARM, ISA.RISCV]:
+            if get_isa_from_str(options.isa) in [ISA.X86, ISA.ARM, ISA.RISCV]:
```

Comment:
Also here I would use the system.cpu[i] type

Reply:
Done